### PR TITLE
Changelog 1.7.7.1

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.7.x
| Description?      | Changelog 1.7.7.1
| Type?             | improvement
| Category?         | PM
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| How to test?      | N/A

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/22550)

### CODE REVIEW

File: docs/CHANGELOG.txt [hunk 1]
```diff
@@ -24,6 +24,72 @@ needs please refer to https://devdocs.prestashop.com/ for more information.
...
+    - #21980: Fixed the error message for a combination with a min quantity for sale (by @Progi1984)
+- Front Office:
+  - Bug fix:
+    - #22296: Center products on category list on mobile (by @NeOMakinG)
+    - #22241: Fix duplicated hook data after product page refresh (by @Hlavtox)
+    - #22103: Display combination image in product pack list (by @jolelievre)
+- Core:
+  - Bug fix:
+    - #22532: Hook keys must be in lowercase (by @PierreRambaud)
+    - #22432: Don't redirect to http from https if it is homepage (Fix Chrome security alert when customer register) (by @ludoc)
```
Comment (matks) [inline docs/CHANGELOG.txt]:
```suggestion
    - #22432: Don't redirect to http from https if it is homepage (by @ludoc)
```
Suggestion (as the changelog aims to explain what has been added into the code): remove issue context, focus on the change

File: docs/CHANGELOG.txt [hunk 2]
```diff
@@ -24,6 +24,72 @@ needs please refer to https://devdocs.prestashop.com/ for more information.
...
+    - #22171: When refreshing an order, display shipping price if needed (by @Progi1984)
+    - #22406: Fixed pagination in Product Quick Navigation Popup (by @Progi1984)
+    - #21977: Add debounce to quantity change of order create product (by @NeOMakinG)
+    - #22291: Fixed Pagination of Sell > Orders > Credits Slips page (by @Progi1984)
+    - #22214: Fix numeric_iso_code in the upgrade to 1.7.7.1 (by @atomiix)
+    - #22126: Fix bug on displayed price's calculation in BO order page under specific conditions (by @matthieu-rolland)
+    - #22246: Integrate ContextStateManager into refund handlers (by @jolelievre)
+    - #21568: Fixed in Create Order the update of cart rules when changing adress  (by @Progi1984)
+    - #22189: Trigger input when a predefined message is applied (by @Progi1984)
+    - #22077: [BC Break] Create Order - Fix remove cart rule when amount is not reached (by @sowbiba)
```
Comment (matks) [inline docs/CHANGELOG.txt]:
```suggestion
    - #22077: Create Order - Fix remove cart rule when amount is not reached (by @sowbiba)
```
Suggestion (as the changelog aims to explain what has been added into the code): remove PR characteristic, focus on the change

**Comment by AliShareei** [comment]
Please merge this #22564 for the next release (1.7.7.1)

**Comment by matks** [comment]
Hi @AliShareei I am sorry but the scope of https://github.com/PrestaShop/PrestaShop/issues/22306 has been defined before and 1.7.71 is actually _already built_ (see status https://github.com/PrestaShop/PrestaShop/issues/22306). It is now awaiting QA before release.

If your PR fixes a bug that is a regression from PS 1.7.6 it will be added to PS 1.7.7.2 .

**Comment by AliShareei** [comment]
Hi @matks 
Yes, unfortunately, This problem has made it impossible for users to install PrestaShop. 
And many of them do not know how to install it because they think Prestashop has a problem. not demo data!
So in my opinion, this problem is very, very important.

File: docs/CHANGELOG.txt (hunk 2 above)
Comment (Progi1984) [reply docs/CHANGELOG.txt]: Fixed

File: docs/CHANGELOG.txt (hunk 1 above)
Comment (Progi1984) [reply docs/CHANGELOG.txt]: Fixed

**Review by matthieu-rolland — DISMISSED** [review]

File: docs/CHANGELOG.txt [hunk 3]
```diff
@@ -24,6 +24,72 @@ needs please refer to https://devdocs.prestashop.com/ for more information.
 Release Notes for PrestaShop 1.7
 --------------------------------
 
+####################################
+#   v1.7.7.1 - (2021-01-XX)
```
Comment (atomiix) [inline docs/CHANGELOG.txt]:
```suggestion
#   v1.7.7.1 - (2021-01-11)
```

Comment (Progi1984) [reply docs/CHANGELOG.txt]: Fixed

**Review by atomiix — DISMISSED** [review]

**Review by kpodemski — DISMISSED** [review]

**Review by matthieu-rolland — DISMISSED** [review]

**Review by jolelievre — CHANGES_REQUESTED** [review]
Branch starts from appropriate commit (a78afb98cdf5a1f3bb31c8b5d6f3a2aa9afc99f3)
Changelog is up to date with last PR committed for 1.7.7.1 (https://github.com/PrestaShop/PrestaShop/pull/22543)

All good for me, I approve the PR but I block it to avoid it from being merged too early, an additional approval must be added

**Review by matks — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Review by PululuK — APPROVED** [review]

**Pull request merged**